src/components/ImageGalleryGrid/index.js

```diff
@@ -6,59 +6,94 @@ import {connect} from 'react-redux'
 import {fetchUserImages as fetchUserImagesAction} from 'src/actions/userImages'
 import ImageThumbnail from '../ImageThumbnail'
 
-class ImageGalleryGrid extends React.Component {
+const ImagePagination = (props) => {
+    // During the first modal load there is no data yet. Data is fetched in ComponentDidMount.
+    if (props.responseMetadata === undefined) {
+        return null;
+    }
+    
+    const pageAmount = Math.ceil(parseInt(props.responseMetadata.count) / 100);
+    const currentPage = props.responseMetadata.currentPage;
+    
+    let classes;
+    const pages = [];
+    
+    for (let i = 1; i < pageAmount + 1; i++) {
+        classes = (props.responseMetadata.currentPage !== undefined && currentPage == i) ? 'page-item active' : 'page-item';
+        
+        pages.push(<li className={classes} key={i}><a className='page-link' href='#' onClick={() => props.clickedPage(i)}>{i}</a></li>);
+    }
 
+    return <nav aria-label='Page navigation example'>
```

Comment:
is the `aria-label` needed and if so, maybe change the text to something more fitting?